### PR TITLE
clusterroleaggregation: use contextual logging

### PR DESCRIPTION
**Pull request by mengjiao-liu**
#### What type of PR is this?
/kind feature

#### What this PR does / why we need it:
clusterroleaggregation: use contextual logging

#### Which issue(s) this PR fixes:

 Part of https://github.com/kubernetes/enhancements/issues/3077

#### Does this PR introduce a user-facing change?

```release-note
Migrated the ClusterRole aggregation controller (within `kube-controller-manager`) to use [contextual logging](https://k8s.io/docs/concepts/cluster-administration/system-logs/#contextual-logging).
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
- [KEP]: https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/3077-contextual-logging
```

### CODE REVIEW

File: cmd/kube-controller-manager/app/rbac.go [hunk 1]
```diff
@@ -19,12 +19,15 @@ package app
 import (
 	"context"
 
+	"k8s.io/klog/v2"
+
```
Comment (pohly) [inline cmd/kube-controller-manager/app/rbac.go:23]: Please remove the blank line.

**Review by pohly — CHANGES_REQUESTED** [review]

File: cmd/kube-controller-manager/app/rbac.go (hunk 1 above)
Comment (mengjiao-liu) [reply cmd/kube-controller-manager/app/rbac.go]: Updated.

**Review by pohly — APPROVED** [review]
/lgtm

**Comment by sftim** [comment]
Suggestion for the release note:
```markdown
Migrated the ClusterRole aggregation controller (within `kube-controller-manager`) to use [contextual logging](https://k8s.io/docs/concepts/cluster-administration/system-logs/#contextual-logging).
```

AIUI, end users could observe this change.

**Comment by mengjiao-liu** [comment]
> Suggestion for the release note:
> 
> ```gfm
> Migrated the ClusterRole aggregation controller (within `kube-controller-manager`) to use [contextual logging](https://k8s.io/docs/concepts/cluster-administration/system-logs/#contextual-logging).
> ```
> 
> AIUI, end users could observe this change.

Comment addressed. Thanks.

**Review by atiratree — APPROVED** [review]

**Comment by pohly** [comment]
/assign @soltysh 

For approval.

**Comment by mengjiao-liu** [comment]
ping @soltysh

**Review by soltysh — APPROVED** [review]
/approve

**Pull request merged**